easybuild/easyconfigs/s/Salome/Salome-8.5.0-intel-2020a.eb

```diff
@@ -0,0 +1,29 @@
+name = 'Salome'
+version = '8.5.0'
+
+easyblock = "Tarball"
+
+homepage = 'https://www.salome-platform.org/'
+description = """Salome is an open-source software that provides a generic Pre- and Post-Processing platform for numerical
+ simulation. It is based on an open and flexible architecture made of reusable components."""
+
+toolchain = {'name': 'intel', 'version': '2020a'}
+
+sources = [{
+    'source_urls': ['https://www.salome-platform.org/downloads/previous-versions/salome-v%(version)s/'],
+    'download_filename':'DownloadDistr?platform=OS1.CO7&version=%(version)s',
+    'filename':'SALOME-%(version)s-CO7-SRC.tgz'
+}]
+
+checksums = ['abacf5e6ac05e6ab17730a05dfd1d6ac5b71cb83fbd62be90c4d05ed79fd1164']
+
+dependencies = [("libGLU", "9.0.1")]
```

Comment:
Many of the dependencies listed in `co7_dep_sys_rpms.txt` will not be necessarily installed in the system. I would add those dependencies to this easyconfig to use packages from EB.

Reply:
@ekieffer you marked the suggestion made by @lexming  as resolved, but you did not actually address it.
it looks like the deps listed in `co7_dep_sys_rpms.txt` are expected to be installed as system centos7 RPM installs. if that the case, that's not good. 
there's also a dir `BINARIES-CO7/` that seems to contain at least some (but not all) required system deps, but it's better to add those as EB dependencies in the easyconfig instead.